RELEASE.md

```diff
@@ -0,0 +1,77 @@
+# Releasing 
+
+- [Releasing](#releasing)
+  - [Before You Begin](#before-you-begin)
+  - [Local Release Preparation](#local-release-preparation)
+    - [Update the CHANGELOG.md](#update-the-changelogmd)
+    - [Create Release Commit and Tag](#create-release-commit-and-tag)
+    - [Run Pre-Release Tests](#run-pre-release-tests)
+  - [Submit Release Pull Request](#submit-release-pull-request)
+  - [Push Release Tag](#push-release-tag)
+  - [Verify Github Release](#verify-github-release)
+  - [Update Homebrew versions](#update-homebrew-versions)
+
+## Before You Begin
+
+Ensure your local workstation is configured to be able to [Sign commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)
+
+## Local Release Preparation
+
+### Checkout latest code
+
+```bash
+git checkout master
+git pull origin master
+```
+
+### Update the [CHANGELOG.md](CHANGELOG.md)
+
+- Add a new version header at the top of the document, just after `# [Unreleased]`
+- Update links at bottom of changelog
+
+### Create Release Commit and Tag
+
+```bash
+export RELEASE_VERSION=2.x.y
+git commit -a -S -m "Release $RELEASE_VERSION"
+git tag -s "v$RELEASE_VERSION" -m "Release $RELEASE_VERSION"
```

Comment:
@neufeldtech Here is a bit different message.

The tag message should be similar to one in Release.

Example:

```
v2.x.y

- Add release doc (#1231, @neufeldtech)
- Oops I did it again (#321, @miry)
```